Form-Controls/styles.css

```diff
@@ -0,0 +1,70 @@
+{
+    margin:0;
+    padding:0
+}
+.body{
+    background-position: center;
```

Comment:
What are you positioning here? There are two background-image rules on this selector but no background image. What were you going for?